### PR TITLE
[#171] make authentication available to all pages

### PR DESCRIPTION
**Pull request by justin-cotarla**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Infrastructure.pdf
 .DS_Store
 .env
-coverage/
\ No newline at end of file
+coverage/
+server/build/
```
Comment (tehblasian) [inline .gitignore:5]: Should this have been pushed?

Comment (justin-cotarla) [reply .gitignore:5]: Yes, coverage is for coverage reports created by jest, and build is generated when a production build is run.

Comment (tehblasian) [reply .gitignore:5]: Alright

**Review by tehblasian — APPROVED** [review]

**Comment by tehblasian** [comment]
Good, this will make authentication great again.

**Pull request merged**